src/computational_graph/diagram.jl

```diff
@@ -80,44 +80,40 @@ Base.:(==)(a::ExternalVertice, b::ExternalVertice) = Base.isequal(a, b)
 - orders::Vector{Int} : orders of the diagram, loop order, derivative order, etc.
 - internal_points::Vector{Int} : internal points in the diagram
 - currents::Vector{Float64} : independent currents in the diagram
-- extVertices::Vector{ExternalVertice}    : external vertices of the diagram
+- external_vertex::Vector{ExternalVertex}    : external vertices of the diagram
```

Comment:
I would call this field itself `external_vertices`. It is a list of the external vertices (i.e., of ExternalVertex objects).

Reply:
fixed